ui/src/app/utils/index.tsx

```diff
@@ -537,26 +537,29 @@ export function validateInputForMySQL(
     inputErrors.add('There is an unclosed " in the search string');
   }
   // check each endsWith term - terms that start with *
-  searchString.split(' ').forEach((word) => {
-    // consecutive special chars
-    if (word.match(/[+*-]{2,}/)) {
-      inputErrors.add(
-        `Search term [${word}] cannot contain consecutive special characters `
-      );
-    }
-    if (word.match(/^\*.*\*$/)) {
-      inputErrors.add(
-        `Search term [${word}] cannot start and end in wild character '*'`
-      );
-    }
-    // length of every word without the special chars mut be >= searchTrigger
-    // for hyphenated word there will be at least 2 letters, if hyphen is removed
-    if (word.replace(/[+*"'-]/g, '').length < searchTrigger) {
-      inputErrors.add(
-        `Search term [${word}] length must be at least ${searchTrigger} characters without special characters`
-      );
-    }
-  });
+  searchString
+    .trim()
```

Comment:
@chenchals Found this while testing the tooltip. If you leave a space on the end, it shows an error message. 
<img width="793" alt="Screen Shot 2022-09-16 at 9 28 21 AM" src="https://user-images.githubusercontent.com/40036095/190664443-ad25ffa8-ac14-46e3-9c08-2e2ad84022d2.png">

Adding `trim()` before validation fixes this. I missed when reviewing #6972 so including here.